### DISCUSSION TITLE
GLFM Dingus and proof of running in WebAssembly...

### DISCUSSION THREAD

**Discussion by digitalmoksha** [kivikakk/comrak#360] [Show and tell]
In my journey with `comrak`, we use `comrak` in a new project, [gitlab-glfm-markdown](https://gitlab.com/gitlab-org/ruby/gems/gitlab-glfm-markdown).  It is meant to specifically render GitLab Flavored Markdown (GLFM). It's young, experimental, and evolving. While it provides a Ruby interface (same concept as `commonmarker`), it allows us to try and move some of our transformations into Rust.

In order to demonstrate that it works in WebAssembly and provide some demo capability, I ported the [CommonMark dingus](https://spec.commonmark.org/dingus/) to use it.

[GitLab Flavored Markdown dingus](https://gitlab-org.gitlab.io/ruby/gems/gitlab-glfm-markdown/)

Kinda cool to see `comrak` running in a browser...

**Comment by kivikakk** [comment]
I love this, thank you so much for sharing! :D

**Comment by kivikakk** [comment]
This is now linked from the [Comrak homepage](https://comrak.ee/#usage) as an example of using the WASM target!
